main.go

```diff
@@ -145,10 +145,11 @@ func main() {
 	}
 
 	if err = controllers.NewReconciler(controllers.Params{
-		Client: mgr.GetClient(),
-		Log:    ctrl.Log.WithName("controllers").WithName("OpenTelemetryCollector"),
-		Scheme: mgr.GetScheme(),
-		Config: cfg,
+		Client:   mgr.GetClient(),
+		Log:      ctrl.Log.WithName("controllers").WithName("OpenTelemetryCollector"),
+		Scheme:   mgr.GetScheme(),
+		Config:   cfg,
+		Recorder: mgr.GetEventRecorderFor("otel-controller"),
```

Comment:
Can we use `opentelemetry-operator`, or do they recommend using the shortest possible name?

Reply:
My bad. Agreed, we should be consistent - would change it to opentelemetry-operator.